tgui/packages/tgui/interfaces/OpposingForcePanel.js

```diff
@@ -0,0 +1,783 @@
+                disabled={!can_submit || blocked}
+                content="Submit Application"
+                onClick={() => act('submit')} />
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                icon="question"
+                color="orange"
+                tooltip={"Request an update from the admins." + (request_updates_muted ? " (Muted)" : "")}
+                disabled={!can_request_update || request_updates_muted}
+                content="Ask For Update"
+                onClick={() => act('request_update')} />
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                icon="wrench"
+                color="blue"
+                tooltip="Modify your application, this will reset all authorisations."
+                disabled={can_edit}
+                content="Modify Request"
+                onClick={() => act('modify_request')} />
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                icon="trash"
+                color="bad"
+                tooltip="Remove your application from the queue."
+                disabled={status === "Not submitted"}
+                content="Withdraw Application"
+                onClick={() => act('close_application')} />
+            </Stack.Item>
+          </Stack>
+          <NoticeBox
+            color={approved ? "good" : denied ? "bad" : "orange"}
+            mt={2}>
+            {status}
+          </NoticeBox>
+        </Section>
+      </Stack.Item>
+      <Stack.Item>
+        <Section title="Backstory">
+          <TextArea
+            disabled={!can_edit}
+            height="100px"
+            value={backstory}
+            placeholder="Provide a description of why you want to do bad things. Include specifics such as what lead upto the events that made you want to do bad things, think of it as though you were your character, react appropriately."
+            onChange={(_e, value) => act('set_backstory', {
+              backstory: value,
+            })} />
+        </Section>
+      </Stack.Item>
+      <Stack.Item>
+        <Section title="Objectives"
+          buttons={(
+            <Button
+              icon="plus"
+              content="Add Objective"
+              onClick={() => act('add_objective')} />
+          )}>
+          { !!objectives.length && (
+            <OpposingForceObjectives />
+          )}
+        </Section>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+export const OpposingForceObjectives = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    objectives = [],
+    can_edit,
+  } = data;
+
+  const [
+    selectedObjectiveID,
+    setSelectedObjective,
+  ] = useLocalState(context, 'objectives', objectives[0]?.id);
+
+  const selectedObjective = objectives.find(objective => {
+    return objective.id === selectedObjectiveID;
+  });
+
+  return (
```

Comment:
These should start opened, it'd just be a lot simpler to review the application this way.